packages/osfamilymap.yaml

```diff
@@ -27,3 +27,12 @@ RedHat:
       pkgs:
         - rubygems
 
+FreeBSD:
+  pkgs:
+    required:
+      pkgs:
+        - devel/py-pip
+        - lang/ruby25
```

Comment:
This package `lang/ruby25` shouldn't be in the dependencies for `gems`, below instead of here?

Reply:
Thanks, is this better?

Reply:
```
FreeBSD:
  pips:
    required:
      pkgs:
        - devel/py-pip
  pkgs:
    required:
      pkgs:
        - lang/ruby25
        - devel/ruby-gems
```

Reply:
I think there's a typo on the second part. It should be

```yaml
FreeBSD:
  pips:
    required:
      pkgs:
        - devel/py-pip
  gems:
    required:
      pkgs:
        - lang/ruby25
        - devel/ruby-gems
```